luminex/src/org/labkey/luminex/LuminexExcelParser.java

```diff
@@ -242,6 +253,93 @@ private void parseFile() throws ExperimentException
+                    // once we recognize that it is a titration. However I found this makes debugging a bit harder / perhaps the code less readable because the description count is
+                    // no longer there/updated and accurate to the data.
+                    mini.put(analyteName, potentialTitrationCounts.get(desc));
+                    crossFilePT.put(desc, mini);
+                    if ((type.contentEquals("raw")) && ((potentialTitrationCounts.get(desc)) >= LuminexDataHandler.MINIMUM_TITRATION_RAW_COUNT))
+                    {
+                        _titrations.put(desc, potentialTitrations.get(desc));
+                    }
+                    else if ((type.contentEquals("summary")) && ((potentialTitrationCounts.get(desc)) >= LuminexDataHandler.MINIMUM_TITRATION_SUMMARY_COUNT))
+                    {
+                        _titrations.put(desc, potentialTitrations.get(desc));
+                    }
+
+                    if (potentialTitrations.get(desc).isQcControl() )
+                    {
+                        if ((type.contentEquals("raw") && (potentialTitrationCounts.get(desc) <= LuminexDataHandler.SINGLE_POINT_CONTROL_RAW_COUNT)) ||
+                            (type.contentEquals("summary") && (potentialTitrationCounts.get(desc) == LuminexDataHandler.SINGLE_POINT_CONTROL_SUMMARY_COUNT)))
+                        {
+                            _singlePointControls.put(desc, new SinglePointControl(potentialTitrations.get(desc)));
+                        }
+                    }
+                }
+            }
+        }
+        return crossFilePT;
+    }
+
+    private void checkCrossFilePTs(Map<String, HashMap<String, Integer>> crossFilePT, Map<String, Titration> crossFilePTs, Map<String, HashSet<Double>> dilutionCounts, String type)
+    {
+        int minimum_titration_count = (type.contentEquals("raw")) ? LuminexDataHandler.MINIMUM_TITRATION_RAW_COUNT : LuminexDataHandler.MINIMUM_TITRATION_SUMMARY_COUNT;
+        int single_point_control_count = (type.contentEquals("raw")) ? LuminexDataHandler.SINGLE_POINT_CONTROL_RAW_COUNT : LuminexDataHandler.SINGLE_POINT_CONTROL_SUMMARY_COUNT;
+
+        for (String desc : crossFilePTs.keySet())
+        {
+            if (crossFilePT.containsKey(desc))
+            {
+                for (String analyte : crossFilePT.get(desc).keySet())
+                {
+                    boolean foundTitration = false;
+                    if ((crossFilePT.get(desc).get(analyte) >= minimum_titration_count) && (dilutionCounts.get(desc).size() >= 3))
+                    {
+                        _titrations.put(desc, crossFilePTs.get(desc));
+                        foundTitration = true;
+                    }
+                    // to reviewer: Realized I didn't ask this. Are cross-plate point controls something that is done? If not, we can remove this chunk.

```

Comment:
No. I don't think so. I believe only titrations would need to be considered for the cross plate checks, so I agree this can be removed.

Reply:
Done!